sys/dev/dpt/Makefile

```diff
@@ -0,0 +1,8 @@
+# $FreeBSD$
+
+.PATH:        ${SRCTOP}/sys/dev/dpt
+
+KMOD=        dpt_pci
+SRCS=        dpt_pci.c bus_if.h device_if.h pci_if.h smbus_if.h
```

Comment:
These things were specific to amdsmb. You don't need smbus_if.h, for example.

Do you know if this works?